src/preset_cli/cli/superset/sync/native/command.py

```diff
@@ -354,10 +362,20 @@ def import_resources_individually(  # pylint: disable=too-many-locals
     with open(log_file_path, "w", encoding="utf-8") as log_file:
         for resource_name, get_related_uuids in imports:
             for path, config in configs.items():
-                if path.parts[1] != resource_name or path in assets_to_skip:
+                if path.parts[1] != resource_name:
                     continue
 
                 asset_configs = {path: config}
+                for uuid in get_related_uuids(config):
+                    asset_configs.update(related_configs[uuid])
+                related_configs[config["uuid"]] = asset_configs
```

Comment:
This was already there -- we just need to make sure it's executed before the asset is skipped to be imported.